### PR TITLE
[DEV-5696] Add reprocess failures

### PR DESCRIPTION
**Pull request by kbard**
**Description:**

Added some minor tweaks to submission loader to automatically handle failure retries and improve logging.

### CODE REVIEW

File: usaspending_api/etl/management/commands/load_submission.py [hunk 1]
```diff
@@ -74,10 +74,13 @@ def handle_loading(self, db_cursor, *args, **options):
         self.skip_final_of_fy_calculation = options["skip_final_of_fy_calculation"]
         self.db_cursor = db_cursor
 
+        logger.info(f"Starting processing for submission {self.submission_id}...")
```
Comment (tony-sappe) [inline usaspending_api/etl/management/commands/load_submission.py:77]: Is there more information on the submission at this point? I realized yesterday I missed the agency name in the log output (originally printed during the ID fetch from Broker, but included the sub load starts would be more helpful)

Comment (kbard) [reply usaspending_api/etl/management/commands/load_submission.py:77]: > but included the sub load starts would be more helpful

Having a hard time parsing this.  Can you use smaller words that my sleep deprived brain my better comprehend?

Comment (tony-sappe) [reply usaspending_api/etl/management/commands/load_submission.py:77]: LOL.

(originally _the names were_ printed during the ID fetch from Broker, but _including them_ at _submission_ load starts would be more helpful)

Comment (kbard) [reply usaspending_api/etl/management/commands/load_submission.py:77]: Couldn't add it this early, but it should appear near the top of the logs now.

Comment (tony-sappe) [reply usaspending_api/etl/management/commands/load_submission.py:77]: thanks a bunch

**Comment by kbard** [comment]
Warmfixing

**Pull request closed without merging**